### DISCUSSION TITLE
strings.Trim()

### DISCUSSION THREAD

**Discussion by Sibyl1122** [cue-lang/cue#2347] [Q&A]
import "strings"
strings.Trim("stellardb-worker-9j269-2","stellardb-worker-9j269-")

cannot get expect anwser. expect "2",got ""

**Comment by verdverm** [comment]
You probably want TrimPrefix

Trim uses a cutset, so given this example, the 2 is expected to be trimmed

**Comment by Sibyl1122** [comment]
got it ,thanks 👍
